judge/pdf_problems.py

```diff
@@ -9,6 +10,11 @@
 
 from django.conf import settings
 from django.utils.translation import gettext
+from selenium import webdriver
```

Comment:
This will just crash on startup if the user doesn't have selenium.

Reply:
Fixed